README.md

```diff
@@ -1,4 +1,4 @@
-# LBH Base API
+# Academy Resident Information API
 
 Base API is a boilerplate code for being reused for new APIs for LBH
```

Comment:
We could probably remove this line 🤔

Reply:
Yeah I just know that we made a ticket to go through all the README's and I didn't want to bring that work into this ticket